pkg/kudoctl/packages/install/install.go

```diff
@@ -0,0 +1,144 @@
+// Package install provides function to install package resources
+// on a Kubernetes cluster.
+package install
+
+import (
+	"strings"
+	"time"
+
+	"github.com/kudobuilder/kudo/pkg/apis/kudo/v1beta1"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/clog"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/kudo"
+)
+
+type Options struct {
+	skipInstance    bool
+	wait            *time.Duration
+	createNamespace bool
+}
+
+type Option func(*Options)
+
+// SkipInstance installs only Operator and OperatorVersion
+// of an operator package.
+func SkipInstance() Option {
+	return func(o *Options) {
+		o.skipInstance = true
+	}
+}
+
+// WaitForInstance waits an amount of time for the instance
+// to complete installation.
+func WaitForInstance(duration time.Duration) Option {
+	return func(o *Options) {
+		o.wait = &duration
+	}
+}
+
+// CreateNamespace creates the specified namespace before installation.
+// If available, a namespace manifest in the operator package is
+// rendered using the installation parameters.
+func CreateNamespace() Option {
+	return func(o *Options) {
+		o.createNamespace = true
+	}
+}
+
+// Package installs an operator package with parameters into a namespace.
+// Instance name, namespace and operator parameters are applied to the
+// operator package resources. These rendered resources are then created
+// on the Kubernetes cluster.
+func Package(
```

Comment:
Shouldn't this be:
```suggestion
func InstallPackage(
```
as with other similar methods e.g. `installNamespace` or `installInstance`?

Reply:
I want to avoid that this is `install.InstallPackage` when called from other packages. OTOH, for the other functions the `install` prefix makes them more readable because they're private and only called in this package.